public/default.css

```diff
@@ -42,9 +54,53 @@ body {
   margin-top: 50px;
 }
 
-#artistDeets {
-  position: relative;
-  font-size: 25px;
-  font-family: 'poiret one', cursive;
+.details {
+  text-align: center;
+  border-bottom: 1px black solid;
+  border-spacing: 100px;
+  margin-right: 25%;
+  margin-left: 25%;
+  font-weight: bold;
+}
+
+.artName {
```

Comment:
Stay away from abbreviations. Don't forget, we use `kebab-case` for CSS classes.